Source/WebCore/Modules/encryptedmedia/MediaKeySystemController.cpp

```diff
@@ -47,7 +47,7 @@ MediaKeySystemController* MediaKeySystemController::from(Page* page)
 }
 
 MediaKeySystemController::MediaKeySystemController(MediaKeySystemClient& client)
-    : m_client(client)
+    : m_client(&client)
```

Comment:
Is this necessary?

Reply:
Hm I can revert this part, it's not strictly necessary. Thanks for the review!